backend/app/infrastructure/database/models.py

```diff
@@ -41,6 +41,17 @@ def get_budget(self, epoch: int) -> Optional[int]:
         return int(budget) if budget is not None else None
 
 
+class GPStamps(BaseModel):
```

Comment:
For the sake of consistency please add `id` column, instead of reusing primary key from `users`

Reply:
:+1: